specification/app/resource-manager/Microsoft.App/preview/2025-02-02-preview/examples/ManagedEnvironmentsStorages_CreateOrUpdate.json

```diff
@@ -31,6 +31,24 @@
         }
       }
     }
-  }
+  },
```

Comment:
@shivamkumar Will it return always 201 or  200 if key vault is not present and 201 if present. 
I think always returning 201 is recommended way

Reply:
Currently(The sync API) returns 200 OK response only. IMO we shouldn't change that.

Also I see the same pattern in other APIs: returning 200 Ok when sync and 201 created when async:
https://github.com/Azure/azure-rest-api-specs/blob/main/specification/app/resource-manager/Microsoft.App/preview/2024-10-02-preview/examples/SessionPools_CreateOrUpdate.json
https://github.com/Azure/azure-rest-api-specs/blob/main/specification/app/resource-manager/Microsoft.App/preview/2024-10-02-preview/examples/ManagedCertificate_CreateOrUpdate.json